src/Commands/Down.php

```diff
@@ -22,24 +22,28 @@ class Down extends DownCommand
 
     public function handle(): int
     {
-        // The base down command is heavily used. Instead of saving the data inside a file,
-        // the data is stored the tenant database, which means some Laravel features
-        // are not available with tenants.
-
+        // First we retrieve the compiled payload with all the
+        // parameters given, then for each tenant we will
+        // put down for maintenance and pass the payload.
```

Comment:
You can probably get rid of the component altogether since the code is self-explanatory